src/cloudant/_common_util.py

```diff
@@ -234,6 +234,28 @@ def codify(code_or_str):
         return _Code(code_or_str)
     return code_or_str
 
+def get_docs(r_session, url, encoder=None, **kwargs):
+    """
+    Provides a helper for functions that require GET or POST requests
+    with a JSON, text, or raw response containing documents.
+
+    :param r_session: Authentication session from the client
+    :param str url: URL containing the endpoint
+    :param JSONEncoder encoder: Custom encoder from the client
+
+    :returns: Raw response content from the specified endpoint
+    """
+    keys_list = kwargs.pop('keys', None)
+    params = python_to_couch(kwargs)
+    resp = None
+    if keys_list:
+        keys = json.dumps({'keys': keys_list}, cls=encoder)
+        resp = r_session.post(url, params=params, data=keys)
+    else:
+        resp = r_session.get(url, params=params)
```

Comment:
You did not address [this](https://github.com/cloudant/python-cloudant/pull/185/files/98b9287bd549dc0e90da11cd55e03e2dfed484ee#r69211986).  Either provide a reason not to do it or make the change.  I'm fine with either.